README.md

```diff
@@ -100,11 +112,11 @@ If you are only developing the web-component you can run the following to start
 
 Then, in another terminal, run the following to serve the web-component:
 
-    npx nx serve web-component
+    npx nx dev-build web-component
```

Comment:
hmm - does `dev-build` actually serve the web-component? I find it a bit unintuitive that a command with the word `build` in it is serving something.

Reply:
My question here, `project.json` already has a `serve` target. Does it not work instead of this new `dev-build`?